packages/playground/json/index.ssr.html

```diff
@@ -0,0 +1,22 @@
+<div class="fetch-json-module">
+  json-module:
+  <pre></pre>
+  <code></code>
+</div>
+<div class="fetch-json-fs">
+  json-fs:
+  <pre></pre>
+  <code></code>
+</div>
+
+<script type="module">
+  const start = Date.now()
+  text('.fetch-json-module pre', await (await fetch('/json-module')).text())
+  text('.fetch-json-module code', Date.now() - start)
+
+  text('.fetch-json-fs pre', await (await fetch('/json-fs')).text())
+  text('.fetch-json-fs code', Date.now() - start)
```

Comment:
Unless I miss something `Date.now() - start` is measuring the time taken to load both json files and not only `/json-fs`?